app/translations/messages.pot

```diff
@@ -598,715 +622,678 @@ msgstr ""
+#: app/templates/static/cookies-privacy.html:142
+#: app/templates/static/cookies-privacy.html:166
 msgid "Expires"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:135
-#: app/templates/static/cookies-privacy.html:140
+#: app/templates/static/cookies-privacy.html:115
+#: app/templates/static/cookies-privacy.html:120
 msgid ""
 "This helps us count how many people visit our website by tracking if "
-"you&rsquo;ve visited before"
+"you’ve visited before"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:136
+#: app/templates/static/cookies-privacy.html:116
 msgid "2 years"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:141
+#: app/templates/static/cookies-privacy.html:121
 msgid "24 hours"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:145
+#: app/templates/static/cookies-privacy.html:125
 msgid "Used to manage the rate at which page view requests are made"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:146
+#: app/templates/static/cookies-privacy.html:126
 msgid "10 minutes"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:154
+#: app/templates/static/cookies-privacy.html:134
 msgid "Completing a survey (eq.ons.gov.uk)"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:167
+#: app/templates/static/cookies-privacy.html:147
 msgid "Session"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:168
+#: app/templates/static/cookies-privacy.html:148
 msgid "To keep you signed in when completing a survey"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:169
+#: app/templates/static/cookies-privacy.html:149
 msgid ""
-"When a user logs out or closes the browser. Or when\n"
-"                                            their cookie cache is cleared"
+"When a user logs out or closes the browser. Or when their cookie cache is"
+" cleared"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:179
+#: app/templates/static/cookies-privacy.html:158
 msgid "Your account"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:193
+#: app/templates/static/cookies-privacy.html:172
 msgid "To check that your browser can support cookies"
 msgstr ""
 
-#: app/templates/static/cookies-privacy.html:194
-#: app/templates/static/cookies-privacy.html:203
-#: app/templates/static/cookies-privacy.html:212
-#: app/templates/static/cookies-privacy.html:239
-#: app/templates/static/cookies-privacy.html:248
-#: app/templates/static/cookies-privacy.html:257
-msgid ""
-"When a user closes the browser or when their cookie\n"
-"                                            cache is cleared"
+#: app/templates/static/cookies-privacy.html:173
+#: app/templates/static/cookies-privacy.html:180
+#: app/templates/static/cookies-privacy.html:187
+#: app/templates/static/cookies-privacy.html:208
+#: app/templates/static/cookies-privacy.html:215
+#: app/templates/static/cookies-privacy.html:222
+msgid "When a user closes the browser or when their cookie cache is cleared"
```

Comment:
I wonder how this page will make sense with each paragraph translated out of context? maybe the page need to be translated as a whole?

Reply:
I agree, but that's not possible with our current translation approach. Also, the lines will all appear together in the csv/spreadsheet that's generated from the .pot file. We've sent that file to the translators to get some feedback on if it will work for them.